src/scaffoldmaker/meshtypes/meshtype_3d_lung1.py

```diff
@@ -461,15 +461,43 @@ def generateBaseMesh(cls, region, options):
                                 upperNodeIds[e3 + 1][e2][e1], upperNodeIds[e3 + 1][e2][e1 + 1], upperNodeIds[e3 + 1][e2 + 1][e1], upperNodeIds[e3 + 1][e2 + 1][e1 + 1]]
 
                             if (e3 < (uElementsCount3 - 1)) and (e2 == (uElementsCount2 - 1)) and (e1 == 0):
-                                # Distal-front wedge elements
+                                # Distal-front wedge elements / lung edge (inside)
                                 nodeIdentifiers.pop(6)
                                 nodeIdentifiers.pop(2)
-                                eft = eftWedgeCollapseXi1_37
+                                # eft = eftWedgeCollapseXi1_37
+                                # ------------------------------
+                                eft = eftfactory.createEftBasic()
+                                setEftScaleFactorIds(eft, [1], [])
+                                nodes = [3, 4, 7, 8]
+                                collapseNodes = [3, 7]
+                                remapEftNodeValueLabel(eft, nodes, Node.VALUE_LABEL_D_DS1, [])
+                                remapEftNodeValueLabel(eft, collapseNodes, Node.VALUE_LABEL_D_DS2, [(Node.VALUE_LABEL_D_DS1, []), (Node.VALUE_LABEL_D_DS2, [])])
+                                ln_map = [1, 2, 3, 3, 4, 5, 6, 6]
+                                # zero cross derivative parameters
+                                remapEftNodeValueLabel(eft, nodes, Node.VALUE_LABEL_D2_DS1DS2, [])
+                                remapEftNodeValueLabel(eft, nodes, Node.VALUE_LABEL_D2_DS1DS3, [])
+                                remapEftNodeValueLabel(eft, nodes, Node.VALUE_LABEL_D3_DS1DS2DS3, [])
```

Comment:
Don't need these lines as I think your eftfactor has already eliminated cross derivatives? Also in 3 places below.